matlab_simulation/05-estimation/ekf_fs.m

```diff
@@ -0,0 +1,28 @@
+function [meanFeatP,covaFeatP,Hmu]=ekf_fs(predFeatPrior,obsFeat,meanFeat,covaFeat,measUncertainty,checkNewFeat)
+  if (checkNewFeat == 1) %nonobserved features are all 1;old features are 0;
+                % If new feature, initialize the features.
+                meanFeatP(1,1) = predFeatPrior(1,1)+obsFeat(1,1)*cos(obsFeat(2,1)+predFeatPrior(3,1));
```

Comment:
If this linearised motion model can be the argument of fastSLAM(), the fastSLAM() function is wonderful.